File: TransCoder_tokenized_test_set_functions/python/MAXIMUM_SUM_ABSOLUTE_DIFFERENCE_ARRAY.py
def MaxSumDifference ( a, n ) :
    np . sort ( a );
    
    j = 0
    finalSequence = [ 0 for x in range ( n ) ]
    for i in range ( 0, int ( n / 2 ) ) :
        finalSequence [ j ] = a [ i ]
        finalSequence [ j + 1 ] = a [ n - i - 1 ]
        j = j + 2
    MaximumSum = 0
    for i in range ( 0, n - 1 ) :
        MaximumSum = ( MaximumSum + abs ( finalSequence [ i ] - finalSequence [ i + 1 ] ) )
    MaximumSum = ( MaximumSum + abs ( finalSequence [ n - 1 ] - finalSequence [ 0 ] ) );
    
    print ( MaximumSum )

File: results/java_codex/MAXIMUM_SUM_ABSOLUTE_DIFFERENCE_ARRAY.java
int MaxSumDifference(int a[], int n) {
  int finalSequence[n];
  sort(a, a + n);
  for (int i = 0; i < n / 2; ++i) {
    finalSequence[i] = a[i];
    finalSequence[n - i - 1] = a[n - i - 1];
  }
  int MaximumSum = 0;
  for (int i = 0; i < n - 1; ++i) {
    MaximumSum = MaximumSum + abs(finalSequence[i] - finalSequence[i + 1]);
  }
  MaximumSum = MaximumSum + abs(finalSequence[n - 1] - finalSequence[0]);
  return MaximumSum;
}